src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3158,11 +3157,9 @@ sub get_nodeflg {
   $proc = "";
   # reset default if applicable
 
-  $ans = query(" Processing nodes (1:Haswell, 2:Skylake, 3:Cascase, 4:Milan)?", "2");
+  $ans = query(" Processing nodes (3:Cascade, 4:Milan)?", "4");
 
-  if    ($ans == 1) { $nodeflg = "hasw"; $ncpus_per_node = 24 }
-  elsif ($ans == 2) { $nodeflg = "sky";  $ncpus_per_node = 36 }
-  elsif ($ans == 3) { $nodeflg = "cas";  $ncpus_per_node = 45 }
+  if    ($ans == 3) { $nodeflg = "cas";  $ncpus_per_node = 45 }
   elsif ($ans == 4) { $nodeflg = "mil";  $ncpus_per_node = 54 }
```

Comment:
Is there a reason to use only 54 cores on mil?  Aren't there ~126 usable cores on each mil node?

Reply:
leave this to Ricardo and Joe to make the call.